README.md

```diff
@@ -23,13 +21,13 @@
     ]
     ```
 
-2. Добавить пакет бандла в проект:
+2. Add the bundle package to the project:
 
     ```bash
     $ composer require creative-packages/selectel-flysystem-bundle
```

Comment:
See composer.json file — not a `creative-packages/selectel-flysystem-bundle` but only `creative/selectel-flysystem-bundle`. That's all

Reply:
update